vm-setup/roles/libvirt/templates/network.xml.j2

```diff
@@ -37,10 +37,16 @@
 {% set numflavor = lookup('vars', 'num_' + flavor + 's')|default(0)|int %}
 {% for num in range(0, numflavor) %}
 {% set ironic_name = ironic_prefix + flavor + "_" + num|string %}
+{% set hostname_format = lookup('vars', flavor + '_hostname_format', default='') %}
+{% if hostname_format == '' %}
+{% set hostname = flavor ~ '-' ~ num %}
```

Comment:
Could we just specify a default of `flavor-%s` in the lookup instead?

Reply:
Ah, yes we can. This logic is a holdover from when I wasn't sure how the value templating was going to work. I'll update this shortly.